Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ComponentEditor.cpp

```diff
@@ -214,8 +211,34 @@ namespace AzToolsFramework
             if (!aggregateInstance)
             {
                 // Set the adapter component to this instance.
-                // Note: multiple selection with DPE is not yet supported
                 m_adapter->SetComponent(componentInstance);
+
+                // set the DPE back to the regular adapter and destroy the aggregate adapter
+                if (m_aggregateAdapter)
+                {
+                    m_dpe->SetAdapter(m_adapter);
+                    m_aggregateAdapter.reset();
+                }
+            }
+            else
+            {
+                if (!m_aggregateAdapter)
+                {
+                    m_aggregateAdapter = AZStd::make_shared<AZ::DocumentPropertyEditor::LabeledRowAggregateAdapter>();
+
+                    // for now, disable "values differ rows", since there are so many pointer and opaque types in the Inspector
+                    // and the output is noisy and unpleasant.
+                    m_aggregateAdapter->SetGenerateDiffRows(false);
+                    m_aggregateAdapter->AddAdapter(m_adapter);
+                    m_filterAdapter->SetSourceAdapter(m_aggregateAdapter);
+                    m_dpe->SetAdapter(m_filterAdapter);
+                }
+
+                AZStd::shared_ptr<AZ::DocumentPropertyEditor::ComponentAdapter> newAdapter =
+                    (Prefab::IsInspectorOverrideManagementEnabled() ? AZStd::make_shared<Prefab::PrefabComponentAdapter>()
```

Comment:
A comment to explain this would be nice. It took me a while to understand what's happening as well. The fact that we could be calling AddAdapter() twice for the aggregate instance tripped me.

Reply:
it won't add the aggregate instance twice, there's always a call with the componentInstance specified first (and aggregateInstance is null) before adding to it with additional calls to AddInstance with the aggregateInstance pointer set. I'll explain further below.

Reply:
added comments.